plugin/adsk/scripts/mayaUsdTranslatorExport.mel

```diff
@@ -15,20 +15,92 @@
+    }
+}
+
+global proc mayausdTranslatorExport_SetTextField(string $arg, string $widget) {
+    textFieldGrp -e -text $arg $widget;
+}
+
+global proc mayausdTranslatorExport_SetOptionMenuByAnnotation(
+        string $ann, string $widget) {
+    int $index = 1; // 1-based indexing.
+    for ($i in `optionMenuGrp -q -itemListLong $widget`) {
+        if (`menuItem -q -ann $i` == $ann) {
+            optionMenuGrp -e -select $index $widget;
+            return;
+        }
+
+        $index++;
+    }
+}
+
+global proc string mayausdTranslatorExport_AppendFromCheckbox(string $currentOptions, string $arg, string $widget) {
+    if (`checkBoxGrp -q -v1 $widget` == 1) {
+        return $currentOptions + ";" + $arg + "=1";
+    } else {
+        return $currentOptions + ";" + $arg + "=0";
+    }
+}
+
+global proc string mayausdTranslatorExport_AppendFromPopup(string $currentOptions, string $arg, string $widget) {
+    int $index = `optionMenuGrp -q -select $widget` - 1;
+    string $allItems[] = `optionMenuGrp -q -itemListLong $widget`;
+    string $item = $allItems[$index];
+    string $annotation = `menuItem -q -ann $item`;
+    return $currentOptions + ";" + $arg + "=" + $annotation;
+}
+
+proc string mayausdTranslatorExport_AppendFrameRange(string $currentOptions) {
+    int $start = `intFieldGrp -q -value1 framRangeFields`;
+    int $end = `intFieldGrp -q -value2 framRangeFields`;
+    return $currentOptions + ";startTime=" + $start + ";endTime=" + $end;
+}
+
+global proc string mayausdTranslatorExport_AppendFromIntField(string $currentOptions, string $arg, string $widget) {
+    string $value = `intFieldGrp -q -value1 $widget`;
+    return $currentOptions + ";" + $arg + "=" + $value;
+}
+
+global proc string mayausdTranslatorExport_AppendFromFloatField(string $currentOptions, string $arg, string $widget) {
+    string $value = `floatFieldGrp -q -value1 $widget`;
+    return $currentOptions + ";" + $arg + "=" + $value;
+}
+
+global proc string mayausdTranslatorExport_AppendFromTextField(string $currentOptions, string $arg, string $widget) {
+    string $value = `textFieldGrp -q -text $widget`;
+    return $currentOptions + ";" + $arg + "=" + $value;
+}
+
 global proc int mayaUsdTranslatorExport (string $parent,
                                      string $action,
                                      string $initialSettings,
                                      string $resultCallback )
 //
 //  Description:
-//    This script posts the mayaUsd export file translator options.
+//    This script posts the USD Export file translator options.
 //    The optionsString is of the form:
 //    varName1=value1;varName2=value2;...
 //
 //  Parameters:
 //    $parent - the elf parent layout for this options layout. It is
 //            always a scrollLayout.
-//    $action - the action that is to be performed with this invocation of this proc.
+//    $action - the action that is to be performed with this invokation of this proc.
```

Comment:
?

Reply:
Hmm...I didn't even notice these two "typo" fixes.  I didn't change those myself but I'll revert them both.

Reply:
And just like the spacing above, these two typos actually came over from the equivalent code in the Pixar plugin.